jmx-metrics/src/main/groovy/io/opentelemetry/contrib/jmxmetrics/JmxConfig.java

```diff
@@ -32,6 +32,16 @@ class JmxConfig {
   static final String JMX_REMOTE_PROFILE = PREFIX + "jmx.remote.profile";
   static final String JMX_REALM = PREFIX + "jmx.realm";
 
+  static final List<String> JAVA_SYSTEM_PROPERTIES =
+      Arrays.asList(
+          "otel.resource.attributes",
+          "javax.net.ssl.keyStore",
+          "javax.net.ssl.keyStorePassword",
+          "javax.net.ssl.keyStoreType",
+          "javax.net.ssl.trustStore",
+          "javax.net.ssl.trustStorePassword",
+          "javax.net.ssl.trustStoreType");
```

Comment:
What is consuming these properties? If it's the metric exporter, should these be copied in the `GroovyMetricEnvironment` together with all `otel.*` props?